polkadot-protocol/protocol-components/accounts/account-balance-types.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Account Balance Types
+description: The Account Balances page provides a comprehensive overview of the various types of balances associated with accounts in the Polkadot ecosystem.
+---
+
+# Account Balance Types
+
+In the Polkadot ecosystem, accounts have different types of balances based on their activity. These balance types determine whether your funds can be used for transfers, to pay fees, or must remain frozen due to specific on-chain requirements.
+
+!!! note "A more efficient distribution of account balance types is in development"
+    Soon, pallets in the Polkadot SDK will implement the [`fungible` trait](https://paritytech.github.io/polkadot-sdk/master/frame_support/traits/tokens/fungible/index.html){target=\_blank} (see the [tracking issue](https://github.com/paritytech/polkadot-sdk/issues/226){target=\_blank} for more details). 
+    
+    This update will enable more efficient use of account balances, allowing the free balance to be utilized for on-chain activities such as setting proxies and managing identities.
+
+There are 5 types of account balances:
+
+- **Free**: The balance available for on-chain activities such as staking and participating in governance, but not necessarily spendable or transferable.
+- **Frozen**: The portion of the free balance that is locked for specific purposes like [staking](path to staking page), [governance](path to governance page), or [vesting](path to Vested Transfers subsection in Types of Txs) (also referred to as locked balance).
+- **On Hold**: The balance reserved for [identities](path to Account Identities section), [proxies](path to Proxy Accounts page), [OpenGov preimages and deposits](path to OpenGov deposits page). It is no longer free and is sometimes called reserved balance.
+- **Spendable**: The free balance that is fully available for spending or transferring.
+- **Untouchable**: The part of the free balance that cannot be transferred or spent, but can still be used for other on-chain activities.
+
+The spendable balance is calculated as follows:
+
+```
+spendable = free - max(frozen - on_hold, ED)
+```
+
+where `free`, `frozen` and `on_hold` are defined above. The `ED` is the the **existential deposit**.
+
+!!! note
```

Comment:
These are the terms that most people know about. In the list above, I think it is better to replace On Hold with Reserve and Frozen with Locked. What do you think?

Reply:
That makes sense, I will amend it